web-server/v0.4/e2etests

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+cd $(dirname $0)
+cat > mock/datastoreConfig.js <<EOF
+export default {
+  '/dev/datastoreConfig': {
+    elasticsearch: 'http://test_domain.com',
+    results: 'http://test_domain.com',
+    graphql: 'http://test_domain.com',
+    prefix: 'test_prefix.',
+    run_index: 'test_index.',
+  },
+};
+EOF
+yarn --network-timeout 1000000
+if [[ $? -ne 0 ]]; then
+  echo "yarn failed!" >&2
+  exit 1
+fi
+yarn start &
```

Comment:
Checking the status using `$?` does not report success or failure of the creation of the background task.  Unless we do the suggestion below, or something like it, I think we can just remove this check entirely.

Reply:
I agree with the proposed changes below, adding a dynamic check would help ensure we can proceed with E2E tests and verify that the local environment has been compiled successfully.